templates/hooks/alias/sdk_read_many_post_build_request.go.tpl

```diff
@@ -1,2 +0,0 @@
-// Match by associated Key ID
-input.KeyId = r.ko.Spec.TargetKeyID
```

Comment:
Why was this removed, though?

Reply:
The reason I ask is because this was previously avoiding the need to paginate...

Reply:
That's what I thought too, but not entirely. There can be multiple aliases for any 1 key. This just filters the number of returned aliases down. There still may be too many to return in a single response - so pagination is still required.

Reply:
Oh `ListAliases` defaults to 50 aliases returned. So there is no need for pagination here. https://docs.aws.amazon.com/kms/latest/APIReference/API_ListAliases.html

Even if the maximum # aliases per key was higher, the API can return up to 100 aliases.

Reply:
>So there is no need for pagination here.

Per Key filter returns NotFound for an existing alias which is not correct and does not allow update path to be triggered.

And when we are not listing per key, there can be many aliases and then pagination will help avoid false positive NotFound errors too.

Reply:
> > So there is no need for pagination here.
> 
> Per Key filter returns NotFound for an existing alias(during update) which is not correct and does not allow update path to be triggered.
> 
> And when we are not listing per key, there can be many aliases and then pagination will help avoid false positive NotFound errors too.

OK, that's just weird :)